tests/e2e/Makefile

```diff
@@ -79,7 +79,7 @@ run-e2e-tests: list-test-suites
 		$(MAKE) run-e2e-tests-$$suite -i; \
 	done
 	# Show a final report
-	junitcli --report reports
+	$(shell go env GOPATH)/bin/junitcli --report reports
```

Comment:
isn it `$(shell pwd)/bin`?

Reply:
Since it is installed using `go get..` it is installed under `GOPATH`

Reply:
Would it make sense to install it into ${PWD}/bin ?

Reply:
If it is possible to install it using `go get...`, we are using that method instead of installing it manually.